src/sim_recon/cli/parsing.py

```diff
@@ -167,6 +167,7 @@ def parse_recon_args(
     parser.add_argument(
         "-o",
         "--output-directory",
+        default=None,
```

Comment:
Why is this necessary? See https://docs.python.org/3/library/argparse.html#default

Reply:
It isn't, I just like being explicit and consistent with the other arguments

Reply:
Then you're missing more code changes!

Reply:
My point is that `default` is missing from the `add_argument` calls above if you want to consistently explicit